app/views/podcasts/show.html.erb

```diff
@@ -42,7 +42,13 @@
         <div class="card-body episode-card h-auto border-start-0">
           <% if @next_scheduled.present? %>
             <div class="episode-card-info">
-              <p class="episode-card-date"><%= @next_scheduled.released_at.to_date.strftime("%m/%d") %></p>
+              <p class="episode-card-date">
+                <% if @next_scheduled.published_at.present? %>
+                  <%= @next_scheduled.published_at.to_date.strftime("%m/%d") %>
```

Comment:
Now that I'm thinking about it - this should use the `l()` localization method. 

```ruby
l(@next_scheduled.published_or_released_at, format: :short)
```

(May need to add/change that format to en.yml too)